Arrangement_on_surface_2/demo/Arrangement_on_surface_2/AlgebraicCurveParser.cpp

```diff
@@ -146,7 +146,7 @@ AlgebraicCurveParser<Polynomial_d>::operator()(const std::string& expression)
     return {};
 }
 
-#ifdef CGAL_USE_CORE
+#ifdef CGAL_USE_Core
```

Comment:
I disagree. In CGAL, there are 189 occurrences of `CGAL_USE_CORE` and 0 of `CGAL_USE_Core`.